src/auro-accordion.js

```diff
@@ -3,15 +3,21 @@
 
 // ---------------------------------------------------------------------
 
+/* eslint-disable lit/binding-positions, lit/no-invalid-html */
+
 // If using litElement base class
-import { LitElement, html, nothing } from "lit";
+import { LitElement, nothing } from "lit";
+import { html } from 'lit/static-html.js';
 import { classMap } from 'lit/directives/class-map.js';
 
-// Import Icons
-import chevronUp from "@alaskaairux/icons/dist/icons/interface/chevron-up.mjs";
-import chevronDown from "@alaskaairux/icons/dist/icons/interface/chevron-down.mjs";
+import { AuroDependencyVersioning } from '@aurodesignsystem/auro-library/scripts/runtime/dependencyTagVersioning.mjs';
+
+import { AuroIcon } from '@aurodesignsystem/auro-icon/src/auro-icon.js';
```

Comment:
We should not do this. When using icons within a component where we know what icons are being used, we should be importing them directly and not using auro-icon that pulls from a CDN we do not have control over.